[{"lineNumber": 3, "col_offset": 0, "nodeName": "exception_factory", "type": "Callable[..., Any]"}, {"lineNumber": 15, "col_offset": 0, "nodeName": "DataSetNotLoadedError", "type": "Type[DataSetNotLoadedError]"}, {"lineNumber": 17, "col_offset": 0, "nodeName": "PartialDependenceError", "type": "Type[PartialDependenceError]"}, {"lineNumber": 19, "col_offset": 0, "nodeName": "DataSetError", "type": "Type[DataSetError]"}, {"lineNumber": 21, "col_offset": 0, "nodeName": "ModelError", "type": "Type[ModelError]"}, {"lineNumber": 38, "col_offset": 0, "nodeName": "MatplotlibUnavailableError", "type": "Type[MatplotlibUnavailableError]"}, {"lineNumber": 39, "col_offset": 0, "nodeName": "MatplotlibDisplayError", "type": "Type[MatplotlibDisplayError]"}, {"lineNumber": 3, "col_offset": 55, "nodeName": "Exception", "type": "Type[Exception]"}, {"lineNumber": 15, "col_offset": 24, "nodeName": "exception_factory", "type": "Callable[..., Any]"}, {"lineNumber": 17, "col_offset": 25, "nodeName": "exception_factory", "type": "Callable[..., Any]"}, {"lineNumber": 19, "col_offset": 15, "nodeName": "exception_factory", "type": "Callable[..., Any]"}, {"lineNumber": 21, "col_offset": 13, "nodeName": "exception_factory", "type": "Callable[..., Any]"}, {"lineNumber": 23, "col_offset": 23, "nodeName": "exception_factory", "type": "Callable[..., Any]"}, {"lineNumber": 26, "col_offset": 25, "nodeName": "exception_factory", "type": "Callable[..., Any]"}, {"lineNumber": 29, "col_offset": 24, "nodeName": "exception_factory", "type": "Callable[..., Any]"}, {"lineNumber": 32, "col_offset": 21, "nodeName": "exception_factory", "type": "Callable[..., Any]"}, {"lineNumber": 35, "col_offset": 26, "nodeName": "exception_factory", "type": "Callable[..., Any]"}, {"lineNumber": 38, "col_offset": 29, "nodeName": "exception_factory", "type": "Callable[..., Any]"}, {"lineNumber": 39, "col_offset": 25, "nodeName": "exception_factory", "type": "Callable[..., Any]"}, {"lineNumber": 5, "col_offset": 19, "nodeName": "__init__", "type": "Callable[..., None]"}, {"lineNumber": 7, "col_offset": 7, "nodeName": "isinstance", "type": "Callable[[Any, Union[Type[Any], Tuple[Union[Type[Any], Tuple[Type[Any], ...]], ...]]], bool]"}, {"lineNumber": 7, "col_offset": 18, "nodeName": "attributes", "type": "None"}, {"lineNumber": 7, "col_offset": 30, "nodeName": "dict", "type": "Type[Dict[Any, Any]]"}, {"lineNumber": 9, "col_offset": 11, "nodeName": "type", "type": "Type[Type[Any]]"}, {"lineNumber": 10, "col_offset": 8, "nodeName": "exception_name", "type": "str"}, {"lineNumber": 12, "col_offset": 8, "nodeName": "attribute_dict", "type": "Dict[str, Callable[..., None]]"}, {"lineNumber": 24, "col_offset": 56, "nodeName": "PartialDependenceError", "type": "Type[PartialDependenceError]"}, {"lineNumber": 27, "col_offset": 58, "nodeName": "PartialDependenceError", "type": "Type[PartialDependenceError]"}, {"lineNumber": 30, "col_offset": 57, "nodeName": "PartialDependenceError", "type": "Type[PartialDependenceError]"}, {"lineNumber": 33, "col_offset": 54, "nodeName": "PartialDependenceError", "type": "Type[PartialDependenceError]"}, {"lineNumber": 36, "col_offset": 59, "nodeName": "PartialDependenceError", "type": "Type[PartialDependenceError]"}, {"lineNumber": 38, "col_offset": 92, "nodeName": "ImportError", "type": "Type[ImportError]"}, {"lineNumber": 39, "col_offset": 84, "nodeName": "RuntimeError", "type": "Type[RuntimeError]"}, {"lineNumber": 5, "col_offset": 19, "nodeName": "base_exception", "type": "Type[Exception]"}, {"lineNumber": 8, "col_offset": 8, "nodeName": "update", "type": "Any"}, {"lineNumber": 8, "col_offset": 26, "nodeName": "attributes", "type": "Any"}, {"lineNumber": 11, "col_offset": 9, "nodeName": "base_exception", "type": "Type[Exception]"}, {"lineNumber": 8, "col_offset": 8, "nodeName": "attributes", "type": "Any"}]